recipes-extended/jailhouse/jailhouse-imx_git.bb

```diff
@@ -13,6 +13,8 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=9fa7f895f96bde2d47fd5b7d95b6ba4d \
                  file://driver/jailhouse.h;beginline=9;endline=36;md5=2825581c1666c44a17955dc574cfbfb3 \
 "
 
+PROVIDES = "virtual/jailhouse"
```

Comment:
Its better to use just `PREFERRED_PROVIDER_jailhouse` and remove the `PROVIDES`

Reply:
> Its better to use just `PREFERRED_PROVIDER_jailhouse` and remove the `PROVIDES`

This doesn't work somehow, didn't have time to dig better into why :(

Reply:
> To utilize virtual here, the original jailhouse recipe must also include it. Another option is to add `jailhouse` to the `PROVIDES` list.

Fixed.

Reply:
I made some code suggestions, please check.

Reply:
> I made some code suggestions, please check.

Now `PREFERRED_PROVIDER_jailhous` solution works :-/ 
It may have been either typo or a cache issue :(

Done.